packages/text-field/src/TextField.js

```diff
@@ -92,6 +92,11 @@ export default class TextField extends Component {
     };
   }
 
+  handleChange = event => {
+    this.setState({ value: event.target.value });
+    return this.props.onChange && this.props.onChange(event);
```

Comment:
That `return` isn't needed, and lets use an if statement instead of an [unused expression](https://eslint.org/docs/rules/no-unused-expressions). The line below is less chars too.

```js
if (this.props.onChange) this.props.onChange(event);
```